core/server/config/env/config.production.json

```diff
@@ -8,5 +8,10 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "logging": {
+        "level": "info",
+        "rotation": true,
+        "transports": ["file"]
```

Comment:
This limitation seems wrong to me and doesn't fit with what is written in the goals on the issue: 
> full logging which outputs to stdout/stderr as we do now (and better!)

If stdout and stderr are different streams, then it would make sense to be able to log them to different places. This is pretty standard across things like nginx and apache, and is also what is currently done on Ghost(Pro). I'm not sure if the existing implementation allows this to be done differently by using a different adapter?